lib/app/features/feed/views/pages/visibility_settings_modal/components/visibility_settings_list_item.dart

```diff
@@ -41,7 +45,7 @@ class VisibilitySettingsListItem extends ConsumerWidget {
         child: option.getIcon(context),
       ),
       trailing: GestureDetector(
-        onTap: () => ref.read(selectedVisibilityOptionsProvider.notifier).selectedOption = option,
```

Comment:
We should be able to select several options before closing the `visibility_setting` bottom sheet, but this change breaks the existing logic